website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -249,17 +264,19 @@ helm install \
 
 ## Updating Appsmith
 
-### Auto Update
+Appsmith installation can be updated manually or by enabling auto update.
+
+### Auto update
 
-*   In the default Appsmith helm installation the auto-update is disabled (Recommended). You have the option to enable auto-update for your Appsmith helm deployment by either:
+The default Appsmith helm installation disables the auto update (_Recommended_). You have an option to enable auto update by either setting it in values file or using a command.
```

Comment:
Should the 'recommended' word be on the heading itself?

Reply:
No, we are not recommending auto-update. The auto-update is disabled by default, which is recommended. Thus, I kept it there only. I didn't add `recommended` to the Manual update and kept it the way it was in the original doc.

Reply:
As discussed, no changes needed here.